pkg/console/subresource/managedclusterview/oauth_server_cert.go

```diff
@@ -12,22 +12,23 @@ import (
 	// managedclusterviewv1beta1 "github.com/open-cluster-management/multicloud-operators-foundation/pkg/apis/action/v1beta1"
 )
 
-func DefaultViewIngressCert(cr *operatorv1.Console, cn string) *unstructured.Unstructured {
-	managedClusterView := ViewIngressCertStub(cn)
-	withDefaultViewIngressCertInfo(managedClusterView, cn)
-	return managedClusterView
-}
-
-func withDefaultViewIngressCertInfo(mcv *unstructured.Unstructured, cn string) {
-	unstructured.SetNestedField(mcv.Object, cn, "metadata", "namespace")
+func DefaultOAuthServerCertView(cr *operatorv1.Console, cn string) (*unstructured.Unstructured, error) {
+	mcv := OAuthServerCertViewStub(cn)
+	err := unstructured.SetNestedField(mcv.Object, api.OAuthServerCertManagedClusterViewName, "metadata", "name")
+	err = unstructured.SetNestedField(mcv.Object, cn, "metadata", "namespace")
+	err = unstructured.SetNestedStringMap(mcv.Object, util.LabelsForManagedClusterResources(cn), "metadata", "labels")
+	if err != nil {
+		return nil, err
```

Comment:
we should return aggregated error, check https://github.com/openshift/console-operator/pull/612#discussion_r787959525